cfpb/consumerfinance.gov
# PfC: nav updates

Prevent popstate bug related to nav and fix nav open state.


## Changes

- Get correct parent when setting open state on nav items
- Only run nav update when there is an active section


## How to test this PR

1. Check out branch
2.  Navigate to [finanancial path tool](http://localhost:8000/paying-for-college/your-financial-path-to-graduation)
3. Check popstate error: click "getting started", then click back button and verify that no JS errors are displayed
4. Check nav: verify that nav appears on mobile, and that sections open and close as expected when navigating through desktop version


## Checklist
 


  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
